webpack/assets/javascripts/react_app/components/DualList/DualList.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DualListControlled as PfDualList } from 'patternfly-react';
+import { arrangeItemsBySelectedIDs } from './DualListHelper';
+
+import './DualList.scss';
+
+const DualList = props => {
+  const { id, items, selectedIDs, inputProps, disabled, error } = props;
+  const { selectedList, unselectedlist } = arrangeItemsBySelectedIDs(
+    items,
+    selectedIDs
+  );
+
+  return (
+    <div id={id} className={`dual-list ${disabled ? 'disabled' : ''}`}>
```

Comment:
`disabled` can be a redux's state from the store, so it can be controlled from other places by dispatching actions.